Client/src/ClientSceneManager/ClientSceneManager.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+** EPITECH PROJECT, 2024
+** R-Type
+** File description:
+** ClientSceneManager
+*/
+
+#include <fstream>
+
+#include "ClientSceneManager.hpp"
+#include "ClientErrors.hpp"
+#include "DLLoader/DLLoader.hpp"
+#include "interfaces/ISceneManager.hpp"
+#include "IComponent.hpp"
+#include "ISystem.hpp"
+
+SceneManager::ClientSceneManager::ClientSceneManager(std::shared_ptr<std::vector<ECS::Registry>> registries)
+{
+    _registries = registries;
+    _registerIndex = SceneManager::RegisterIndex::CURRENT;
+
+    ECS::Registry reg;
+    _registries->push_back(reg);
```

Comment:
Why did you push_back a register ? You already push one in the Application constructor.

Reply:
Well seen